docs/examples/frameworks/jax/pax-basic_example.ipynb

```diff
@@ -0,0 +1,1482 @@
+{
```

Comment:
<p> </p><p>Next we create a helper function <s>to </s>that print<strong>s</strong> training accuracy from <s>the </s>logs. <strong>The </strong>logs created (...)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/5060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Done</p>